0_RootFS/Go/build_tarballs.jl

```diff
@@ -15,13 +15,12 @@ using BinaryBuilder
 include("../common.jl")
 
 name = "Go"
-version = v"1.19"
-version_str = "1.19" # interpolating v1.19 results in "1.19.0"
+version = v"1.22.0"
```

Comment:
I think we may want to keep something like
```julia
version_str = "1.22.0"
version = VersionNumber(version_str)
```
in case they do again something fishy with the trailing zero.

Reply:
Done.